runtime/v1/impl/src/main/java/com/walmartlabs/concord/runner/ConcordSecurityManager.java

```diff
@@ -60,7 +60,7 @@ public void checkPermission(Permission perm) {
             String path = perm.getName();
 
             // allow all local paths
-            if (!path.startsWith("/") && !path.equals(ALL_FILES_TOKEN)) {
+            if (!path.startsWith("/") && !ALL_FILES_TOKEN.equals(path)) {
```

Comment:
Cannot be null here.

Reply:
The purpose of this change is to be proactively defensive. Though it may be negligibly faster to run this way, it also conforms to a best practice and makes the code easier to read. Also, since the behavior is otherwise equivalent, there is no cost to this change.